features/landing-page/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import GitHubIcon from '@mui/icons-material/GitHub';
+import { SupportForm } from 'features/support-form/components/SupportForm';
+import { useTranslation } from "next-i18next";
+
+export default function Footer() {
+
+    const { t } = useTranslation("common");
+
+    return (
+        <footer 
+        className="w-full 
+        max-w-[1200px]
+        h-[250px] 
+        mt-[100px] 
+        pt-[20px] 
+        flex 
+        justify-between 
+        border-t-[0.5px] 
+        border-white/40 
+        font-sans
+        flex-wrap
+        relative
+        
+        max-[768px]:flex-col
+        max-[768px]:items-center
+        max-[768px]:justify-even
+        max-[768px]:h-[300px]">
+            <p 
+            className="text-[24px] 
```

Comment:
i think you're overrusing arbitrary values in few places
why not use `text-2xl`?